OmsiHook/OmsiFileSpline.cs

```diff
@@ -1,14 +1,133 @@
+        {
+            get => Memory.ReadMemory<uint>(Address + 0x28);
+            set => Memory.WriteMemory(Address + 0x28, value);
+        }
+        public uint PrevIDCode
+        {
+            get => Memory.ReadMemory<uint>(Address + 0x2c);
+            set => Memory.WriteMemory(Address + 0x2c, value);
+        }
+        public float EndRadius
+        {
+            get => Memory.ReadMemory<float>(Address + 0x30);
+            set => Memory.WriteMemory(Address + 0x30, value);
+        }
+        /// <summary>
+        /// Euler spiral ??
+        /// </summary>
+        public bool Klothoide
+        {
+            get => Memory.ReadMemory<bool>(Address + 0x34);
+            set => Memory.WriteMemory(Address + 0x34, value);
+        }
+        public byte AlignTerrain
+        {
+            get => Memory.ReadMemory<byte>(Address + 0x35);
+            set => Memory.WriteMemory(Address + 0x35, value);
+        }
+        public float Steig_Start
+        {
+            get => Memory.ReadMemory<float>(Address + 0x38);
+            set => Memory.WriteMemory(Address + 0x38, value);
+        }
+        public float Steig_End
+        {
+            get => Memory.ReadMemory<float>(Address + 0x3c);
+            set => Memory.WriteMemory(Address + 0x3c, value);
+        }
+        public float Cant_Start
+        {
+            get => Memory.ReadMemory<float>(Address + 0x40);
+            set => Memory.WriteMemory(Address + 0x40, value);
+        }
+        public float Cant_End
+        {
+            get => Memory.ReadMemory<float>(Address + 0x44);
+            set => Memory.WriteMemory(Address + 0x44, value);
+        }
+        public float Skew_Start
+        {
+            get => Memory.ReadMemory<float>(Address + 0x48);
+            set => Memory.WriteMemory(Address + 0x48, value);
+        }
+        public float Skew_End
+        {
+            get => Memory.ReadMemory<float>(Address + 0x4c);
+            set => Memory.WriteMemory(Address + 0x4c, value);
+        }
+        public bool useDeltaHeight
+        {
+            get => Memory.ReadMemory<bool>(Address + 0x50);
+            set => Memory.WriteMemory(Address + 0x50, value);
+        }
+        public float DeltaHeight
+        {
+            get => Memory.ReadMemory<float>(Address + 0x54);
+            set => Memory.WriteMemory(Address + 0x54, value);
+        }
+        public bool Mirror
+        {
+            get => Memory.ReadMemory<bool>(Address + 0x58);
+            set => Memory.WriteMemory(Address + 0x58, value);
+        }
+        public OmsiSplineSegment MySegment => new(Memory, Memory.ReadMemory<int>(Address + 0x5c));
+        public MemArray<OmsiFileSplinePathInfoInternal, OmsiFileSplinePathInfo> Paths => new(Memory, Memory.ReadMemory<int>(Address + 0x60));
+        public int Chrono_Origin
+        {
+            get => Memory.ReadMemory<int>(Address + 0x64);
+            set => Memory.WriteMemory(Address + 0x64, value);
+        }
+
```

Comment:
Blanks lines between properties break convention. (well technically blank lines is the convention, but nothing else in the project has blank lines between the properties)